Linear_Search/Linear_Search.ts

```diff
@@ -0,0 +1,36 @@
+/*    Implenetation of LinearSearch algorithm to find value 5 in a given array of numbers    */
+
+function LinearSearch( arr:number[] , n:number ,  val:number ) :number
```

Comment:
please remove the space before comma 
remove the space around parenthesis

Reply:
is it okay now @asha15